hugolib/site.go

```diff
@@ -1355,7 +1356,7 @@ func (s *Site) buildSiteMeta() (err error) {
 
 	s.assembleMenus()
 
-	s.Info.LastChange = s.Pages[0].Lastmod
+	s.Info.LastChange = s.Pages.ByLastmod()[ln-1].Lastmod
```

Comment:
And please add a remark about "work in progress" or a question or something if it is a pull request that needs further work etc.